one_fm/overrides/employee.py

```diff
@@ -111,7 +111,10 @@ def inform_employee_id_update(self):
             sendemail(recipients=[reports_to], subject=subject, content=msg)
 
     def get_reports_to_user(self):
-        return get_employee_reports_to_user(self)
+        approver = get_approver(self.name)
+        if approver:
+            return frappe.db.get_value('Employee', approver, 'user_id')
+        return False
```

Comment:
@pjamsheer Ideally this should return None in case no reports_to is found for an employee instead of returning False.

Reply:
Thank you for your suggestion buddy, I will revert it ASAP